StripeFinancialConnections/StripeFinancialConnections/Source/Native/AccountPicker/AccountPickerViewController.swift

```diff
@@ -133,6 +133,7 @@ final class AccountPickerViewController: UIViewController {
                     let accounts = accountsPayload.data
                     let shouldSkipAccountSelection = accountsPayload.skipAccountSelection ?? self.dataSource.authSession.skipAccountSelection ?? false
                     if !accounts.isEmpty {
+                        // the API is expected to never return 0 accounts
```

Comment:
does this mean `!accounts.isEmpty` is always true? if so should we use `guard` here?

Reply:
This PR does not modify this logic (so this logic is not related to the PR)

Nevertheless, there is an `if accounts.isEmpty` that handles this below. 

Interesting note for context: generally this logic was mostly a direct "copy paste" of Web/Stripe.js to ensure that we have the same logic. We have an experiment where Native is directly 'competing' against Web conversion. For example, even if I believed the code is better structured as X, I still left it as Y to minimize chances of regressions (a change of structure can accidentally cause an unexpected problem).